test/functional/terminal/buffer_spec.lua

```diff
@@ -298,6 +298,18 @@ describe(':terminal buffer', function()
     feed_command('put a') -- buffer a is empty
     helpers.assert_alive()
   end)
+
+  it('supports OSC7 with autochdir', function()
+    command('set autochdir')
+    local cwd = eval("getcwd()")
+    local parent = cwd:match("^(.+\\)")
+    if parent == nil then
+        parent = cwd:match("^(.+/)")
+    end
+    command('terminal')
+    feed('printf "\\033]7;file://host/' .. parent .. '..\\033\\\\"<CR>')
```

Comment:
1. `printf` is shell-dependent
2. You feeded the keys in Normal mode

Reply:
Looking at `test/functional/terminal/helpers.lua` it seems `chansend()` (a.k.a. `jobsend()`) should be used.

Reply:
Thank you so much for your advice, it took me a while but thanks to it I managed to find the right sequence of bytes to make the test pass!

Reply:
I doubt whether OSC 7 makes sense on Windows. I hardly know anything about terminal on Windows though.

Reply:
I managed to fix the test on windows. Thanks again for the advice!